src/components/widgets/sales-cycle/sales-cycle.tmpl.html

```diff
@@ -4,7 +4,7 @@
     <h4>Widget settings</h4>
 
     <div setting-organizations parent-widget="widget" class="part" deferred="::orgDeferred" />
-    <div setting-params-picker options="statusOptions" param="status_selection" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
+    <div setting-params-picker options="statusOptions" param="status_average_durations" has-reach="hasReach" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
```

Comment:
Why changing `param="status_selection"` here? Have you been able to test this widget?